Make workspaces tasks default…

vdemeester opened the pull request:
- Rename tasks with pipeline resources with `-pr`
- Rename tasks with workspace dropping `-workspace` suffix
- Updates README(s), task names, …

Related to https://issues.redhat.com/browse/SRVKP-824

/cc @chmouel @piyush-garg @savitaashture @pradeepitm12 @siamaksade @sthaha @nikhil-thomas @khrm 

Signed-off-by: Vincent Demeester <vincent@sbr.pm>

chmouel commented:
Do we need to keep the -pr if they are not "favoured" ? isnt that going to be kinda confusing ?

vdemeester commented:
> Do we need to keep the -pr if they are not "favoured" ? isnt that going to be kinda confusing ?

the `-pr` wasn't there before, and instead we had `-workspace` on the workspaces ones. I think we can keep the `-pr` for existing user that are using / want to use PipelineResource. @siamaksade wdyt ?

vdemeester committed:
Make workspaces tasks default…

- Rename tasks with pipeline resources with `-pr`
- Rename tasks with workspace dropping `-workspace` suffix
- Updates README(s), task names, …

Signed-off-by: Vincent Demeester <vdemeest@redhat.com>

siamaksade commented:
@vdemeester agreed. We can remove the -pr ones in the next one or two releases

piyush-garg commented on task/s2i-dotnet-1-pr/0.1/s2i-dotnet-1.yaml:
```diff
@@ -1,16 +1,16 @@
 apiVersion: tekton.dev/v1beta1
```
File name should be s2i-dotnet-1-pr.yaml

piyush-garg replied:
This applies for all the  *-pr tasks

piyush-garg commented on task/s2i-dotnet-1/0.1/README.md:
````diff
@@ -62,34 +53,70 @@ oc adm policy add-scc-to-user privileged -z pipeline
 ```
 
-## Creating the taskrun
+## Using a a `Pipeline` with `git-clone`
````
nit: remove one `a`

piyush-garg commented on task/s2i-dotnet-1/0.1/s2i-dotnet-1.yaml:
```diff
@@ -6,11 +6,11 @@ metadata:
 spec:
   description: >-
-    s2i-dotnet-1 task fetches a Git repository and builds and
+    s2i-dotnet-1-workspace task clones a Git repository and builds and
```
need to remove  `-workspace`

nikhil-thomas reviewed (approved):
/lgtm

removing `-worspace` tag from the workspace tasks in this catalog makes sense.

However, when we add them to the operator, we might want to specify the `-workspace` suffix and `version`. Otherwise it might break user pipelines using our ClusterTasks (which use `pipelineResource` by default`).
